projects/packages/forms/src/contact-form/class-contact-form-field.php

```diff
@@ -799,9 +811,26 @@ public function render_textarea_field( $id, $label, $value, $class, $required, $
 	 * @return string HTML
 	 */
 	public function render_radio_field( $id, $label, $value, $class, $required, $required_field_text ) {
-		$field  = '<fieldset id="' . esc_attr( "$id-label" ) . '" class="grunion-radio-options">';
+		$options_classes   = $this->get_attribute( 'optionsclasses' );
+		$options_styles    = $this->get_attribute( 'optionsstyles' );
+		$form_style        = $this->get_form_style();
+		$is_animated_style = 'animated' === $form_style;
+		$fieldset_id       = "id='" . esc_attr( "$id-label" ) . "'";
+
+		if ( $is_animated_style ) {
```

Comment:
Pretty dodgy. Either this, or add the notched HTML to animated style multi fields in the editor. The less of that the better I guessed.

TODO: test this in trunk-created forms. Might need to gate it for inner block forms only.